### DISCUSSION TITLE
nonlinear absorption model

### DISCUSSION THREAD

**Discussion by Shixing-Ju** [Open-Systems-Pharmacology/Forum#1800] [Questions & Problems] [answered]
Hi everyone
I have a drug that has non-linear absorption, and the AUC increase is lower than the linear ratio when the dose is greater than 100mg. How can this be achieved in PK-Sim? If anyone can provide assistance, I would be extremely grateful!

Thanks！

**Comment by Proen1** [comment, marked as answer]
Hi Shixing-Ju , the non-linear absorption can be due to many reasons. Is the chemical in the solution or a specific formulation? Are the higher doses getting closer to the water solubility limit? If we exclude the solubility/dissolution, the remaining option is that you have a membrane transporter. The most pragmatic way is to add a transporter (lumen->enterocytes) and fit the Vmax and Km. The saturation of such a transporter could be what causes the non-linearity. You need to be careful to see if the transporter influx into the cells is not causing an abnormal accumulation in the enterocytes.. maybe compare the accumulation of chemical in enterocytes in the simple passive model and model with active transporter..

**Reply by Shixing-Ju** [reply]
Hi Proen
Thank you very much for your suggestion, which has given me great inspiration.
